content/docs/1.5/scalers/artemis.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "ActiveMQ Artemis"
+layout = "scaler"
+availability = "v1.5+"
+maintainer = "Community"
+description = "Scale applications based on ActiveMQ Artemis"
+go_file = "artemis_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `artemis` trigger for ActiveMQ Artemis.
+
+```yaml
+triggers:
+- type: artemis
+  metadata:
```

Comment:
Can we configure username & password here as well similar to other scalers?

Those should only be pointers to env vars on the workload. We have this as a convenient method for those who don't want to use trigger auth.

Reply:
Ok sure.  I was just directing potential users to better practices. 😁

Reply:
Hehe I fully agree but we can only guide them and provide full docs 😅